src/badguy/granito.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef GRANITO_H
+#define GRANITO_H
```

Comment:
Wrong header guard naming, also missing license info. (Same in granito.cpp)

Reply:
On the coding side, this PR seems to be very WIP so reviewing it now may be pointless.

Reply:
Yeah I know, but I want to point out the obvious things.

Reply:
still thanks for warning. Qt creator usually just does default

Reply:
There's also other issues I notice on the surface, but would keep those for when it's ready for review.

Reply:
How has this never been fixed

Reply:
Nvm github.dev fooled me